molgenis-security-core/src/main/java/org/molgenis/security/acl/ObjectIdentityService.java

```diff
@@ -0,0 +1,21 @@
+package org.molgenis.security.acl;
+
+import java.util.List;
+import java.util.Set;
+import org.springframework.security.acls.model.ObjectIdentity;
+import org.springframework.security.acls.model.Sid;
+
+public interface ObjectIdentityService {
```

Comment:
Please add docs about what this interface does.
Could you add the exceptions that this class thows in case a e.g. classId doesn't exist?
What is the behavior of the Integer parameters and return values? Can they be null? If yes then mark the parameters with @Nullable @CheckForNull. If yes, then change the return values to Optional<Integer>. If no, then use int instead of Integer.